src/api/AllRecipes/AllRecipes.ts

```diff
@@ -1,14 +1,13 @@
-import {api} from "../api.ts";
+import { api } from '../api.ts';
 
 const getAllRecipes = async (dish: string) => {
+  try {
+    const apiKey = import.meta.env.VITE_API_KEY;
```

Comment:
move it out of the function and use UPPER_CASE for constants

const API_KEY = import.meta.env.VITE_API_KEY;

const getAllRecipes = async (dish: string) => {
.....